pkg/endpoints/credentials_test.go

```diff
@@ -501,3 +507,19 @@ func (r Resource) getK8sCredential(namespace string, name string) (credential cr
 	}
 	return secretToCredential(secret,false)
 }
+
+// Get check service account secret
+func (r Resource) checkSecretInSa(namespace string, nameSa string, nameSecret string, t *testing.T) bool {
```

Comment:
Formatting off here, should be using tabs (tab size 2)

Reply:
I can not see the issue.  The format looks OK for me.

Reply:
Was referring to the entire method, but I'll take it!